lasagne/layers/helper.py

```diff
@@ -334,10 +341,12 @@ def get_all_params(layer, **tags):
 
     Notes
     -----
-    If any layer's parameter was set to a Theano expression instead of a shared
-    variable, the shared variables involved in that expression will be returned
-    rather than the expression itself. Tag filtering considers all variables
-    within an expression to be tagged the same.
+    If any of the layer's parameters was set to a Theano expression instead
```

Comment:
Minor nitpick: `If any layer's parameter` allows multiple layers, `If any of the layer's parameters` assumes a single layer (as in `Layer.get_params`). Please either restore the first variant or use yet another option: `If any of the layers' parameters`.